dotcom-rendering/src/web/components/AdSlot.tsx

```diff
@@ -459,6 +459,33 @@ export const AdSlot = ({
 				/>
 			);
 		}
+		case 'mobile-front': {
+			return (
+				<div
+					id={`dfp-ad--mobile-front`}
+					className={[
+						'js-ad-slot',
+						'ad-slot',
+						`ad-slot--mobile-front`,
+						'ad-slot--container-inline',
+						'ad-slot--rendered',
+					].join(' ')}
+					css={[
+						css`
+							position: relative;
+							min-height: 274px;
+							min-width: 300px;
+							width: 300px;
+							margin: 12px auto;
+						`,
+						adStyles,
+					]}
+					data-link-name={`ad slot mobile-front`}
+					data-name={`mobile-front`}
```

Comment:
I was having a look at mobile ads on Frontend on the /uk front, and it seems like all of the ads there (though I could be looking at the wrong ones) come through with the data-link-name as `ad slot inlineX` (where X is the index on the page) and data name `inlineX`. 

Are we purposefully changing the name here (if so have we discussed this with @guardian/commercial-dev) & should we be including the index in the naming scheme to ensure ad slots don't get mixed up?